console/console-init/ui/src/modules/iot-project-detail/ProjectDetailPage.tsx

```diff
@@ -24,13 +19,29 @@ const breadcrumb = (
     <BreadcrumbItem isActive={true}>IoT Project</BreadcrumbItem>
   </Breadcrumb>
 );
-export default function IoTProjectDetailPage() {
+export default function ProjectDetailPage() {
```

Comment:
I will suggest name as IoTProjectDetailPage as it will create confusion in messaging and iot projects